utils/sort-challengefiles.test.js

```diff
@@ -17,7 +17,7 @@ describe('sort-files', () => {
     it('should sort the objects into html, css, jsx, js order', () => {
       const sorted = sortChallengeFiles(challengeFiles);
       const sortedKeys = sorted.map(({ fileKey }) => fileKey);
-      const expected = ['indexhtml', 'indexcss', 'indexjsx', 'indexjs'];
+      const expected = ['indexhtml', 'stylescss', 'indexjsx', 'scriptjs'];
```

Comment:
Definitely do not want `scriptjsx`?

Reply:
Isn't `index.jsx` more common?

Reply:
I do not know 🤷‍♂️ . I use `index` everything.

Reply:
`app.jsx` maybe? 🤷🏻‍♂️

Reply:
Not the point of this PR. So, I am resolving for now.